### ISSUE TITLE
Removal of non-redundant fileprivate

### ISSUE THREAD

**Issue by JensAyton** [nicklockwood/SwiftFormat#814]
In the following code:
```swift
private struct Foo {}

public struct Bar {
    fileprivate let consumeFoo: (Foo) -> Void
}

public func makeBar() -> Bar {
    Bar { _ in }
}
```
the `fileprivate` is rewritten to `private`. This makes the implicit initializer private and `makeBar` becomes invalid.

This is related to, but distinct from ce98a1da06c06df39b7c1e6f322a7f56788fac6b.

**Comment by JensAyton** [comment]
This workaround runs into the same issue:
```swift
private struct Foo {}

public struct Bar {
    private let consumeFoo: (Foo) -> Void

    fileprivate init(consumer: @escaping (Foo) -> Void) {
        consumeFoo = consumer
    }
}

public func makeBar() -> Bar {
    Bar { _ in }
}
```

**Comment by nicklockwood** [comment]
@JensAyton fixed in 0.47.5.

**Comment by JensAyton** [comment]
Thanks, Nick!
